states/massachusetts/elections.toml

```diff
@@ -8,7 +8,87 @@ date = 2020-09-01
 name = "Congressional Primary"
 original_date = 2020-09-01
 
+[20200901.registration]
+in_person_by = 2020-08-22
+online_by = 2020-08-22T23:59:00
+postmarked_by = 2020-08-22
+sources = [
+  "http://www.sec.state.ma.us/ele/eleifv/howreg.htm",
+  "https://www.sec.state.ma.us/ele/elepdf/2020-Primary-and-Election-Schedule-Legal-Sized.pdf",
+]
+# Note the Day to Register was extended to 10 days from 20 days prior in 2020
+
+[20200901.absentee.application]
+email_by = 2020-10-31T12:00:00
+fax_by = 2020-10-31T12:00:00
+received_by = 2020-10-31T12:00:00
```

Comment:
SHould these all set back? "If you are voting by mail, you should be sure to submit your application in a timely manner. Please allow enough time for the ballot to be mailed to you and for you to return the ballot by Election Day. All ballots being mailed from inside the United States must be received by your local election official no later than the close of polls on Election Day."

Reply:
I'd only suggest in_person_by be this date because that implies you actually went in.

Reply:
I'm not super passionate about either way... 

> "If you would like to vote in person, you may do so in the office of your local election official. In-person absentee voting should be arranged ahead of time."

So you could email and then go down to the office...

I would say we're manually changing the data and losing the true answer, so to avoid that, I'll capture the "real-date" as a comment or at least mention we padded it and make the change you suggest.